### ISSUE TITLE
unknown layer type

### ISSUE THREAD

**Issue by kelulucaipeixi** [tonghe90/textspotter#19]
hello @tonghe90 ，
when I run python test.py --img=./imgs/img_105.jpg, it happens like below:

I0902 18:58:18.874758 37749 net.cpp:129] Top shape: 1 1 128 128 (16384)
I0902 18:58:18.874763 37749 net.cpp:137] Memory required for data: 711786500
I0902 18:58:18.874770 37749 layer_factory.hpp:77] Creating layer iou_maps_angles
F0902 18:58:18.874795 37749 layer_factory.hpp:81] Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: Python (known types: AbsVal, Accuracy, AffineTransformer, ArgMax, AttLstm, BNLL, BatchNorm, BatchReindex, Bias, Concat, ContrastiveLoss, Convolution, CosinangleLoss, Crop, Data, Deconvolution, Dropout, DummyData, ELU, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, Input, LRN, LSTMNew, LSTMUnit, Log, Lstm, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Parameter, PointBilinear, Pooling, Power, RNN, ROIPooling, ReLU, Reduction, Reshape, ReverseAxis, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, SmoothL1Loss, Softmax, SoftmaxWithLoss, Split, Sum, TanH, Threshold, Tile, Transpose, UnitboxLoss, WindowData)
*** Check failure stack trace: ***
Aborted (core dumped)

Can you give me a suggestion about how to deal with it? Thank you very much!

**Comment by tonghe90** [comment]
@kelulucaipeixi Did you uncomment  WITH_PYTHON_LAYER := 1 in the Makefile.config

**Comment by kelulucaipeixi** [comment]
@tonghe90 , no, I haven't. This is part of my Makefile.config below:

# Uncomment to support layers written in Python (will link against Python libs)
# WITH_PYTHON_LAYER := 1

# Whatever else you find you need goes here.
INCLUDE_DIRS := $(PYTHON_INCLUDE) /usr/local/include /usr/include/hdf5/serial
LIBRARY_DIRS := $(PYTHON_LIB) /usr/local/lib /usr/lib /usr/lib/x86_64-linux-gnu /usr/lib/x86_64-linux-gnu/hdf5/serial

**Comment by kelulucaipeixi** [comment]
@tonghe90 , there is '#' before the sentence of WITH_PYTHON_LAYER:=1, but the website canceled it.

**Comment by tonghe90** [comment]
@kelulucaipeixi Makefile.config.example is just an example, you have to modify it according to your environment and setting. As we use python layer, so you have to uncomment it to support this.

**Comment by kelulucaipeixi** [comment]
@tonghe90 , oh I'm sorry, I didn't look at your readme carefully, and forget to uncomment it! Thank you very much!
